src/equations/numerical_fluxes.jl

```diff
@@ -261,6 +261,23 @@ function Base.show(io::IO, d::DissipationLaxFriedrichsEntropyVariables)
     print(io, "DissipationLaxFriedrichsEntropyVariables(", d.max_abs_speed, ")")
 end
 
+@doc raw"""
+    MatrixDissipationWintersEtal()
+
+Creates the Roe-like entropy stable matrix dissipation operator from Winters et al (2017). This operator
+must be used together with an entropy-conservative two-point flux function (e.g., `flux_ec`) to yield 
+an entropy-stable surface flux. The surface flux function can be initialized as:
+```julia
+flux_es = FluxPlusDissipation(flux_ec, MatrixDissipationWintersEtal())
+```
```

Comment:
````suggestion
must be used together with an entropy-conservative two-point flux function 
(e.g., [`flux_ranocha`](@ref) or [`flux_chandrashekar`](@ref)) to yield 
an entropy-stable surface flux. The surface flux function can be initialized as:
```julia
flux_es = FluxPlusDissipation(flux_ec, MatrixDissipationWintersEtal())
```
````

`flux_ec` is not defined for the Euler equations, is it?